common/src/main/scala/explore/model/ExploreClipboard.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.effect.IO
+import explore.model.LocalClipboard.*
+import org.scalajs.dom
+
+object ExploreClipboard:
+  private val storageKey = "clipboard"
+
+  def get: IO[LocalClipboard] = IO {
```

Comment:
This doesn't really have to be `IO`. I think everywhere we use it eventually wants a `Callback`. I just more or less recreated the interface of the previous `exploreClipboard`.